maven-model-builder/src/main/java/org/apache/maven/model/building/DefaultModelBuilder.java

```diff
@@ -559,6 +557,10 @@ else if ( isNotEmpty( activationFile.getMissing() ) )
         }
     }
 
+    public static boolean isNotEmpty(String string) {
```

Comment:
I think this can be private. Less API is less to support in the future.

Reply:
Totally agree. I made it private.